resume.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=User resume actions
+After=hibernate.target
```

Comment:
Does this mean it gets triggered only after resume or after cold boot as well?

Reply:
This service only gets triggered after resume. I have tested this very thoroughly.